bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/LogLevel.java

```diff
@@ -0,0 +1,9 @@
+package org.openhab.binding.openthermgateway.internal;
```

Comment:
remove, not needed if you use the a logger per class

Reply:
The connector runs on a separate thread, not sure it has access to the logging framework of openhab itself at runtime. Thats the reason I relay the log messages from the connector thread to the main thread and have it log through openhab logging.